sound/hda/intel-sdw-acpi.c

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+// Copyright(c) 2015-2021 Intel Corporation.
+
+/*
+ * SDW Intel ACPI scan helpers
+ */
+
+#include <linux/acpi.h>
+#include <linux/export.h>
+#include <linux/module.h>
+#include <linux/soundwire/sdw_intel.h>
+#include "../../drivers/soundwire/cadence_master.h"
+#include "../../drivers/soundwire/intel.h"
```

Comment:
wouldn't it be better to move those headers or at least their required parts to include/?

Reply:
yes, it's one of the opens. I need to check first if the destination for the code is agreeable to others before pruning these files to include only what's needed.

Reply:
turns out only one QUIRK is needed, so I moved it to include/linux/soundwire/sdw_intel.h